src/main/webapp/college-info.html

```diff
@@ -30,7 +30,7 @@ <h1 id="school-name"></h1>
         <p id="race-piechart"></p>
         <p id="gender-piechart"></p>
       </section>
-      <a class="reviews-button" href="comments.html">Student Reviews</a>
+      <a class="reviews-button" id="reviews-button">Student Reviews</a>
```

Comment:
What is the reason for changing href="comments.html" to id="reviews-button"?

Reply:
href gets added dynamically in javascript so that it can pas the proper id and load the appropriate comments